stargz/fs.go

```diff
@@ -425,6 +416,17 @@ func (fs *filesystem) check(ctx context.Context, l *layer) error {
 	return nil
 }
 
+func (fs *filesystem) Unmount(ctx context.Context, mountpoint string) error {
+	fs.layerMu.Lock()
+	if _, ok := fs.layer[mountpoint]; !ok {
+		fs.layerMu.Unlock()
+		return fmt.Errorf("specified path %q isn't a mountpoint", mountpoint)
+	}
+	delete(fs.layer, mountpoint) // unregisters the corresponding layer
+	fs.layerMu.Unlock()
+	return syscall.Unmount(mountpoint, 0)
```

Comment:
ditto

Reply:
It is possible that some of the server goroutines become not responding so let's use MNT_FORCE to abort the FUSE connection (https://www.kernel.org/doc/html/latest/filesystems/fuse.html#aborting-a-filesystem-connection). In the future, we might be able to consider to kill that specific hanging goroutine using channel, etc.
But let's don't care about test codes now.